app/components/account/browse/account-dropdown.component.ts

```diff
@@ -46,4 +49,28 @@ export class AccountDropDownComponent implements AfterViewInit {
             this.currentAccountInvalidError = error;
         });
     }
+
+    public openContextMenu(account: AccountResource) {
+        const items = [
```

Comment:
if not enabled just return(don't show context menu) instead of having all enabled set to false